src/sentry/static/sentry/app/views/settings/organization/general/organizationSettingsForm.old.jsx

```diff
@@ -36,13 +37,20 @@ const OldOrganizationSettingsForm = React.createClass({
       openMembership: data.openMembership,
       allowSharedIssues: data.allowSharedIssues,
       isEarlyAdopter: data.isEarlyAdopter,
+      require2FA: data.require2FA,
       enhancedPrivacy: data.enhancedPrivacy,
       dataScrubber: data.dataScrubber,
       dataScrubberDefaults: data.dataScrubberDefaults,
       scrubIPAddresses: data.scrubIPAddresses,
       safeFields: data.safeFields.join('\n'),
       sensitiveFields: data.sensitiveFields.join('\n'),
     };
+
+    //Only for adding the Flag to 2FA Enforcement. Please remove when the feature is released to the public.
+    if (!this.getFeatures().has('require-2fa')) {
+      delete result.require2FA;
```

Comment:
Can we enforce this on at the API level instead?

Reply:
So Billy all I want to do here is create a temporary measure and stop the checkbox from appearing on the frontend. This will remain here only until the feature is finished. Then it will be removed. I just didn't want to have a checkbox in settings that was not implemented.

Reply:
Ah sorry, I thought this was when saving the form. Is this needed since below we hide the BooleanField

Reply:
It might not be. I'm just not familiar with Javascript. I can remove it, if that causes no issues. :)